contracts/crowdsale/Crowdsale.sol

```diff
@@ -193,6 +194,6 @@ contract Crowdsale {
    * @dev Determines how ETH is stored/forwarded on purchases.
    */
   function _forwardFunds() internal {
-    wallet.transfer(msg.value);
+    wallet.safeTransfer(msg.value);
```

Comment:
This should be a regular `transfer` because it is transferring ether. :slightly_smiling_face:

Reply:
Sorry for the stupid suggestion @dougiebuckets :laughing: 
Thanks for catching it @frangio.

Could that be the cause of the weird error?